### ISSUE TITLE
Fman startup ~3sec

### ISSUE THREAD

**Issue by krupitskas** [fman-community/fman#740]
Hey!
I have a laptop
Intel Core i7 8550U 1800MHz/15.6"/1920x1080/16GB/256GB SSD Xiaomi.
In power-saving mode fman startup takes around 3 sec. 
For example Sublime Text around ~0.7.
It's just soo long (imagine you sit and wait when your file browser opens, 1 second pass, 2nd second pass, 3rd-second pass).
Explorer opens with same speed as sublime.
Any ideas?
btw the problem is exacerbated by the fact that you can't call same instance of fman but it starts new one.
P.S. I've just bought fman for support anyway because it still awesome :)

**Comment by Arty2** [comment]
Same here unfortunately, but have no problem opening multiple instances.
Also just bought to support the developer, it's a great, and unique product even with its shortcomings of age.

**Comment by mherrmann** [comment]
Hi and thank you both for buying a license to support me.

Could it be that it's slow because there are many files in the folder that is initially displayed when you open fman?

**Comment by krupitskas** [comment]
To be honest, I've tried different folders with low/empty/really lot of files and folders. It's always pretty the same. I afraid this can be a python bootstrap in your case but I don't know.
The performance of such apps is crucial.
For example, I've bought Sublime Text and don't use VSCode only because of open and usage performance (however it really lack of tiny UI features).

**Comment by ragingtom** [comment]
I agree. The startup time is very slow. Far Manager on the same machine/same folder starts instantly and fman takes 2-3 sec.

**Comment by Shurna83** [comment]
Hello, I recently bought fman (really a great product!).
Also for me startup speed could be improved, it takes ~3-4 secs.

**Comment by dspasicpdy** [comment]
bump; startup takes approx 4 seconds on 10.14.6 Mojave, as well.

**Comment by cja-github** [comment]
I have a top spec XPS and fman takes between 2 and 3 seconds to start, whereas Explorer is pretty much instant. I'd be happy for fman to launch at Windows startup and just show a new or existing window whenever it's called. (I'm a paying user and fman is ace!)
